Readme.md

```diff
@@ -58,7 +58,9 @@ jobs:
       apikey: ${{ secrets.DEFAULT_NUGET_PUBLISH_API_KEY }}
 ```
 
-The _publish.yml_ workflow is triggered on a tag pushed to any branch with the prefix `v` and should contain a [valid version number](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-basics), e.g. `v1.0.1`, which will be extracted and used to version the NuGet packages produced.
+The _publish.yml_ workflow is triggered on a tag pushed to any branch with the prefix `v` and should contain a [valid version number](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-basics), e.g. `v1.0.1`, which will be extracted and used to version the Orchard Core extension manifests and the NuGet packages produced.
+
+There is no configuration required for automated Orchard Core extension manifest versioning, all of the `Manifest.cs` files are looked up, and the existing `Version` properties are updated automatically inside the `Module` definition with the version pushed.
```

Comment:
I think "no configuration required" is not technically true, because `ModuleAttribute.Version` must be provided in the source code for the regex to update it. `Version` is not a required property and if it's not set, then `ModuleAttribute.DefaultVersionZero` ("0.0") is implied. I'd rather write that the `Version` property must be set to a dummy value, e.g. `0.0.1`. That would also document why it's changed to that for all of our submodules.